### PR TITLE
Update "25.4 Using the App-AutoScaler Service"

### LINKED ISSUE #529: App Autoscaler Section Review and Verification
Upstream docs have been updated with new descriptions for features and now uses (more) commands from the plugin rather than from the vanilla CF CLI.  

See https://github.com/cloudfoundry/app-autoscaler/blob/develop/docs/Readme.md

### LINKED ISSUE #645: [doc] 25.4  Using the App-AutoScaler Service
25.4  Using the App-AutoScaler Service

https://documentation.suse.com/suse-cap/1.5.1/html/cap-guides/cha-cap-app-autoscaler.html#sec-cap-app-autoscaler-usage

The $AS_PASSWORD is empty

    peter@caasadmin:~/autoscaler> SECRET=$(kubectl get pods --namespace scf \
    --output jsonpath='{.items[?(.metadata.name=="api-group-0")].spec.containers[?(.name=="api-group")].env[?(.name=="INTERNAL_CA_CERT")].valueFrom.secretKeyRef.name}')
    peter@caasadmin:~/autoscaler> echo $SECRET
    secrets-2.19.1-1
    peter@caasadmin:~/autoscaler> AS_PASSWORD="$(kubectl get secret $SECRET --namespace scf --output jsonpath="{.data['autoscaler-service-broker-password']}" | base64 --decode)"
    peter@caasadmin:~/autoscaler> echo $AS_PASSWORD

    peter@caasadmin:~/autoscaler>

### PR DESCRIPTION
**Pull request by KucharczykL**
Closes #529.
Closes #645.

~~Would be nice to solve #645 as part of this PR as well.~~

### CODE REVIEW

**Comment by btat** [comment]
CAP-251 should contain the info needed to resolve #645

**Comment by KucharczykL** [comment]
I believe the description should now reflect the current way of doing things.

File: xml/cap_admin_app_autoscaler.xml [hunk 1]
```diff
@@ -220,128 +224,107 @@
...
-  <para>
-   Create a new instance of the App-AutoScaler service:
-  </para>
-
-<screen>&prompt.user;cf create-service autoscaler autoscaler-free-plan <replaceable>service_instance_name</replaceable></screen>
-
-  <para>
-   A name of your choice may be used to replace
-   <literal>service_instance_name</literal>.
+   Attach auto-scaling policy to the application:
```
Comment (btat) [inline xml/cap_admin_app_autoscaler.xml]:
```suggestion
   Attach a autoscaling policy to the application:
```

Comment (btat) [reply xml/cap_admin_app_autoscaler.xml]: looks like other instance of autoscaling don't use a `-`

Comment (KucharczykL) [reply xml/cap_admin_app_autoscaler.xml]: You are right, thanks for pointing it out!

**Comment by it-slav** [comment]
Is there somewhere I can read the updated doc?

**Comment by btat** [comment]
Backported to `maintenance/1.4.1` and `maintenance/1.5`

**Pull request merged**